### ISSUE TITLE
[ML Migration] comms to be drafted for kubernetes-dev migration 

### ISSUE THREAD

**Issue by parispittman** [kubernetes/community#6273]
#5877 - umbrella issue 

comms to be drafted [help us with this! comment below with suggestions for each]

To Send:   
**~Jan 2nd**
5: welcome email post migration 
send from dev@kubernetes.io

Subject: dev@kubernetes.io: welcome to the new home for Kubernetes contributing discussions

Body:
Contributors, 

Please update your contact lists to dev@kubernetes.io (old: kubernetes-dev@googlegroups.com). There is nothing else you need to do at this time. 

In 2022, we are going to continue our mission for better communication across the project. Migrating this list to a new @kubernetes.io was one of our (sig-contribex) initiatives and you'll see many more from other groups like release and Steering Committee this year. 

Happy emailing,
sig-contribex

--------------------------------

Sent:
**ASAP & ~Jan 2nd**
4: banner for kdev with migration and sunset details. there is a character limit. 
  - ASAP Before migration: As of Jan 2, 2022 this group will be sunset in favor of a new @kubernetes.io domain. Please read emails for information about this. Members will be migrated.  
  - Jan 2nd After migration: Mailing list for upstream Kubernetes contributors. This includes process changes, release info, contributor events, and more. Try #kubernetes-users on slack or discuss.k8s.io for end user troubleshooting and discussion.

**Dec 8th. (done: sent on the 9th)**
1: to kdev about intentions and timeline

Subject: [Announce] kubernetes-dev@ (this list) is moving to [name-tbd]@kubernetes.io 
Body:
tldr - 
- kubernetes-dev@googlegroups.com to be migrated to <foo>@kubernetes.io on jan 2, 2022
- users will be migrated over, you will not need to resubscribe
- vote on new name by x date here [link]

This mailing list started 7 years ago in 2014 before the project was donated with [this first email](https://groups.google.com/g/kubernetes-dev/c/xAmfbGBhNVA/m/BuSJALFLE_EJ). Since then, the upstream community has grown exponentially: we have hundreds of lists/groups like this for SIGs, their subprojects, alerts, and more. This group has 5747 members and now breaks calendars, gets its fair share of spam (thanks mods), included in a spiderweb of shared docs, mounds of informative release info, and so much valuable contributor news.

SIG Contributor Experience would like to migrate this mailing list (kubernetes-dev@googlegroups.com) to a Kubernetes domain [name]@kubernetes.io. [This umbrella issue](https://github.com/kubernetes/community/issues/5877) has many related issues, justifications, and more context. This list will join steering@, conduct@, and other official communication channels of the project that allow us to configure via pull request and do fun things with an API under the Google Workspace. You can weigh in on the naming convention by emoji reaction on this issue [link to issue].

This list is not being deleted. Members(you) will be migrated(copied) to the new address as of 5PM PT / 1AM UTC January 2, 2022. The banner will be updated alerting new members to join the [new]@kubernetes.io list. We call this a soft sunset: no new postings or members, not deleted. 

_What does this mean for you?_ 
Next email you draft to kubernetes-dev@ after **January 2nd**, use the new address - to be announced. 
Leads of groups: Check out the leads channels for your actions. 

A one week reminder will be sent when we are there. Please comment on the umbrella issues with questions and comments. 

**Dec 8th/9th**
2: leads with actions they need to take and when

Leads,
Following up on the previous thread with intentions of migrating kubernetes-dev@ to a kubernetes.io domain. [link to kdev email]

Actions:
1 - due Monday at Noon PT
vote on naming convention: 
we're gathering consensus on the naming convention here; you and all of your group members are welcome to participate. 

2 - due January 30th 
add the new group to your docs: 
add the winner of the above poll, [newgroup]@kubernetes.io, to all of your most active and viewed shared documents. You can start as early as the announcement but try to be wrapped up no later than January 30th.
- add with viewing only; your sig mailing list and contributors@kubernetes.io: editors
- example of "most active and viewed shared documents": meeting agendas, open KEP collaboration docs, most recent/release planning spreadsheets, your last update slide deck for KubeCon (Los Angeles, etc) 

FAQ:
1 - What about our calendar events?
Those should be owned by the community groups shared calendar under your group-name-leads@ list and not an individual or k-dev. https://github.com/kubernetes/community/blob/master/communication/calendar-guidelines.md#establishing-a-new-meeting 

2 - We have a lot of docs. Help?
If contributors@kubernetes.io is an editor of your docs already, we are happy to help. Let us know in #sig-contribex. 

3 - Should we forward open discussion from k-dev to the new address?
Yes. After scanning the list now, it looks like there might be ~10 open items that would still have active discussion at the time of migration. We're keeping track of this here: https://github.com/kubernetes/community/issues/6275 

4 - If the list isn't being deleted. What's happening?
We are turning off the ability to reply to current postings and post new messages. New membership will also be turned off and messages on the banner of the group will reflect the we've moved. 

**~Dec 27th**
3: 1 week to go migration update to kdev
  - [fwd announcement email] 
  - Reminder: one week to go. 
  - Leads, let #sig-contribex know if you need help adding new group to your docs.
  - dev@kubernetes.io is the new list; members will be migrated

/sig contributor-experience

**Comment by chris-short** [comment]
Are we going to do any forwarding/duplication for a short period after the migration?

I'm concerned about the level of effort and timing of that effort. Maybe there's a way to shortcut the timing. I'd encourage leads to add the new list to docs before it's officially live because there are going to be brand new contributors who won't know about ye ole list or conversely only know about ye ole list.

**Comment by parispittman** [comment]
@chris-short no duplication needed. the list isn't being deleted - only new members won't be able to be added and no new messages will be able to be sent.

**Comment by cblecker** [comment]
Wording in the Dec 8th e-mail mostly LGTM. One question:
The date Jan 2 is used in one place and Jan 3 in another.. which date is it?

**Comment by parispittman** [comment]
last email just went out - we're done here 😎 

/close
